test/e2e/framework/metrics/metrics_grabber.go

```diff
@@ -43,6 +45,8 @@ const (
 	kubeControllerManagerPort = 10257
 	// snapshotControllerPort is the port for the snapshot controller
 	snapshotControllerPort = 9102
+	// kubeProxyPort is the default port for the kube-proxy status server.
+	kubeProxyPort = 10249
```

Comment:
:slightly_frowning_face: I don't know why they are redefining them here rather than using the values in `pkg/cluster/ports`...

Reply:
most probably because nobody knew, I didn't know at least